frontend/test/lit/test_pauli_rot_and_measure.py

```diff
@@ -392,9 +394,9 @@ def test_pauli_rot_and_measure_with_cond():
     @qjit(pipelines=pipeline, target="mlir")
     @qml.qnode(device=dev)
     def circuit():
-        qml.PauliRot(np.pi / 2, "Z", wires=0)
+        partial(qml.PauliRot, pauli_word="Z")(np.pi / 2, wires=0)
```

Comment:
It actually doesn't affect normal usage of PauliRot. Only when it is used in conjunction with `qml.cond`.